app/views/staff/training_sessions/new.html.erb

```diff
@@ -5,6 +5,10 @@
   <% label_tag 'training_id' %>
   <%= select_tag 'training_id', options_from_collection_for_select(Training.all, 'id', 'name'), prompt: 'select training' %><br>
 
+  <% if @user.admin? %>
+    <% label_tag 'user_id' %>
+    <%= select_tag 'user_id', options_from_collection_for_select(User.where('role LIKE \'staff\' or role LIKE \'admin\''), 'id', 'name'), prompt: 'select instructor' %><br>
```

Comment:
I would add a scope to the user for this to select admin and staff. Also why do you need `LIKE`? Doesn't `=` work?

eg: `User.where(role: ['staff', 'admin'])`?